modules/iam/users/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "usernames" {
+  description = "Desired names for the IAM user"
+  type        = list(string)
+}
+
+variable "path" {
+  description = "Desired path for the IAM user"
+  default     = "/"
```

Comment:
can be moved to place where it is used as well

Reply:
I moved `path` to local instead in ed73315 ✅